apps/client/src/components/organisms/panel/Panel.component.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactElement, useEffect, useState } from 'react';
+import { Description, Text } from '../..';
+import { ReactComponent as ArrowHeadTop } from 'pixelarticons/svg/chevron-up.svg';
```

Comment:
Rename to `ArrowHeadTopIcon`

Reply:
fixed https://github.com/the-ai-team/razor/pull/41/commits/6609129235ab29039445a1013b4a2c9e3e9da0e8